packages/cypress/package.json

```diff
@@ -57,6 +58,7 @@
     "p-map": "^4.0.0",
     "semver": "^7.5.2",
     "sha-1": "^1.0.0",
+    "stack-utils": "^2.0.6",
```

Comment:
this should catch such issues early:
https://github.com/replayio/replay-cli/blob/78f5c72a62a38772cad90cccf1283b80eea49b61/scripts/pkg-build/src/plugins/resolveErrors.ts#L29-L35

I wonder why it didn't let us know about it here... I'll investigate tomorrow